agreements/employment.md

```diff
@@ -0,0 +1,167 @@
+
+|**Full name**|<textarea></textarea>|
+|**Address**|<textarea></textarea>|
+|**Job title**|<textarea></textarea>|
+|**Job description**|<textarea class="big"></textarea>|
+|**Pay**|<textarea>£xx,000 per annum (pro rata fixed part time or variable part time)</textarea>|
+|**Location**|<textarea class="big">The first place you will be required to work for us under this contract will be Unit B7, 8-9 Hoxton Square, Greater London, N1 6NU, United Kingdom. You may be required to work, either temporarily or permanently, at any office within the Greater London area.</textarea>|
+|**Contract date**|<textarea></textarea>|
+|**Commencement date**|<textarea></textarea>|
+|**Previous continuous employment**|<textarea></textarea>|
+
+<div style="page-break-after:always"></div>
+
+## Terms of your employment
+
+1. ### Employer
+    1. You are employed by The Dextrous Web Ltd, a company registered in England and Wales under number 6617101, trading as dxw. Our registered office is at Unit B7, 8-9 Hoxton Square, Greater London, N1 6NU.
+
+2. ### Date of commencement
+    1. The date for the purposes of calculating your statutory period of continuous employment with the company is your date of commencement, unless otherwise specified in the above table. This takes into account any unbroken employment with us, or any associated company. Service with your previous employer does not count as part of your continuous period of employment with us.
+
+3. ### Pay
+    1. You will be paid monthly in arrears by transfer to your bank or building society account on the last weekday of each calendar month.
+    2. You must always maintain a suitable bank or building society account into which payment can be made, and provide us with current details of it. Changes to your bank or building society details must be notified to us in writing.
+    3. We reserve the right to deduct an amount from any payment due to use from us, or to recover the amound directly from you, if:
+        1. there is an overpayment of wages, expenses or any other sum paid to you during your employment;
+        2. you are paid expenses which are subsequently disallowed;
+        3. there is an outstanding amount due on any loan or advance paid to you by us;
+        4. we become liable for any damage, loss of property, fine or charge as a result of your negligence, dishonesty, breach of contract or failure to follow the company's policies.
+
+4. ### Hours of work
+    1. Unless you work part-time, your normal working day is seven hours long, making a 35 hour working week. During busy periods, you may be required to work longer hours, up to a maximum of 50 hours a week, on average, over a rolling six week period.
+    2. Your normal working hours are 10:00am to 6:00pm with an hour off for lunch. We are open to arrangements for flexible working hours, at the company's sole discretion.
+    3. We do not operate any overtime scheme nor have any arrangements for the paying of bonuses.
```

Comment:
The "nor have any arrangements for the paying of bonuses" seems out of place in this section.